versions.props

```diff
@@ -1,13 +1,13 @@
-com.fasterxml.jackson.*:jackson-* = 2.6.7
+com.fasterxml.jackson.*:jackson-* = 2.8.9
```

Comment:
Any reason not to use the latest 2.9.8?

Reply:
This one matches the dropwizard BOM - I didn't want to impose any unnecessarily high dependencies just because this is so low in the stack.

https://github.com/dropwizard/dropwizard/blob/v1.1.3/dropwizard-bom/pom.xml#L27

Reply:
Sounds reasonable, thanks!